roles/ceph-facts/tasks/facts.yml

```diff
@@ -303,9 +303,18 @@
         chrony_daemon_name: chrony
       when: ansible_os_family == "Debian"
 
-- name: set grafana_server_addr fact
+- name: set grafana_server_addr fact - ipv4
   set_fact:
-    grafana_server_addr: "{{ (hostvars[groups[grafana_server_group_name][0] if groups.get(grafana_server_group_name, []) | length > 0 else groups[mgr_group_name][0]])['ansible_all_ipv4_addresses'] | ipaddr(public_network) | first }}"
+    grafana_server_addr: "{{ (hostvars[groups[grafana_server_group_name][0] | default(groups[mgr_group_name][0])])['ansible_all_ipv4_addresses'] | ipaddr(public_network) | first }}"
```

Comment:
this reintroduce a bug that was just fixed

Reply:
sorry, I correctly rebased the PR now.